openapi/components/schemas/LegacyOneTimeSale.yaml

```
@@ -0,0 +1,215 @@
+      Billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  websiteId:
+    x-sortable: true
+    x-basic: true
+    allOf:
+      - $ref: ./WebsiteId.yaml
+  currency:
+    description: Currency of the order.
+    readOnly: true
+    $ref: ./CurrencyCode.yaml
+  initialInvoiceId:
+    description: ID of the initial invoice (`null` for one-time orders).
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  items:
+    description: Details of items in the order.
+    type: array
+    minItems: 1
+    items:
+      $ref: ./OrderItem.yaml
+  deliveryAddress:
+    description: Delivery address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  billingAddress:
+    description: Billing address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  activationTime:
+    description: Date and time when the order is activated.
+    x-sortable: true
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  voidTime:
+    description: Date and time when the order is voided.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  abandonTime:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      Date and time when the pending order is automatically abandoned.
+      If this value is not passed during order creation,
+      a [pending order TTL](https://www.rebilly.com/catalog/all/Organizations/PatchOrganization/#!t=request&path=settings/billing/pendingOrderTtl) setting is used to calculate the value.
+    format: date-time
+  couponIds:
+    type:
+      - 'array'
+      - 'null'
+    description: |-
+      List of coupons to redeem on the customer and restrict to this order.
+
+      For more information, see [Coupons](https://www.rebilly.com/docs/settings/coupons-and-discounts/).
+
+      This parameter uses the following logic:
+
+      - If this parameter is not supplied, applied coupons are not changed.
+      - If an empty array is supplied, all applied coupon redemptions are canceled.
+      - If a list of coupons is supplied, unapplied coupons in the list are applied.
+        Coupons that have already been applied do not change state.
+        Applied coupons that are not supplied in list are canceled.
```

nitpick: possible typo
```suggestion
        Applied coupons that are not supplied in the list are canceled.
```